cypress/integration/Home/index.spec.ts

```diff
@@ -105,5 +105,29 @@ describe('Home page', () => {
         cy.findByText('Love to hear from you').should('be.visible');
       });
     });
+
+    describe('click on survey item link', () => {
+      it('redirects user to survey detail page', () => {
```

Comment:
```suggestion
      it('redirects user to the survey detail page', () => {
```

Reply:
Updated in [34a5526](https://github.com/hanam1ni/react-surveyor/pull/75/commits/34a5526aa6061380d83efb2e799b89da5a745cee)